server/db/seeds/10-shopProducts.js

```diff
@@ -0,0 +1,22 @@
+exports.seed = async function (knex) {
+  // Deletes ALL existing entries
+  await knex('shop_products').del()
+  await knex('shop_products').insert([
+    {
+      id: 1,
+      name: 'large veggie box',
+      description:
+        'a large box filled with fresh seasonal produce grown at your local garden',
+      price: 29,
```

Comment:
Probably a good idea to use a decimal price to test the data type here, eg: 29.99.